### ISSUE TITLE
Genesis Parameters

### ISSUE THREAD

**Issue by faddat** [eve-network/eve#180]

**Comment by alanwilhelm** [comment]
Looks fresh and clean. Will cosmwasm be completely open?

**Comment by faddat** [comment]
yep this is a totally permissionless cosmwasm chain.  Please feel free to jump in to any of it with your thoughts :)
